gcloud/orb.yaml

```diff
@@ -229,5 +229,95 @@ jobs:
           paths:
             - prev_version/<<parameters.zone>>
 
+  deploy-cloud-run:
+    docker:
+      - image: google/cloud-sdk:alpine
+    parameters:
+      creds:
+        default: GCLOUD_SERVICE_KEY
+      cluster:
+        default: ''
+        description: >
+          If set, deploys to a Knative cluster rather than using the managed
+          Cloud Run service.
+        type: string
+      deployment:
+        type: string
+      # TODO: Find a better way to do this. CircleCI does not have a list
+      # type yet: https://ideas.circleci.com/ideas/CCI-I-701
+      env_vars:
+        type: string
+      image:
+        type: string
+      memory:
+        default: 128Mi
+        type: string
+      project:
+        description: >
+          Name of GCP project to which we will push.
+        type: string
+      region:
+        type: string
+    steps:
+      - auth:
+          creds: <<parameters.creds>>
+          project: <<parameters.project>>
+      - run: echo y | gcloud components install beta
+      - unless:
+          condition: <<parameters.cluster>>
+          steps:
+            - run: |
+                gcloud beta run deploy <<parameters.deployment>> \
+                  --platform=managed \
```

Comment:
Lets add `--no-allow-unauthenticated` as a default here -- looks like @harsh-dialpad was just about to add that [here](https://github.com/talkiq/tooling/pull/173/files)

Reply:
done 👍